### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by jessicawyz**
Add assert and logging
Add small bug fix for prefix checking

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddExamCommand.java [hunk 1]
```diff
@@ -38,7 +40,7 @@ public class AddExamCommand extends Command {
             + PREFIX_TIME + "09:00";
 
     public static final String MESSAGE_SUCCESS = "Added exam to person with ID: %1$s";
-
+    private static final Logger logger = Logger.getLogger(AddExamCommand.class.getName());
```
Comment (MarcusGitty) [inline src/main/java/seedu/address/logic/commands/AddExamCommand.java:43]: The other Logger class just writes the class name and don't have a get name method. maybe wanna change it to follow their format ?

image

File: src/main/java/seedu/address/logic/commands/DeleteExamCommand.java [hunk 2]
```diff
@@ -38,7 +40,7 @@ public class DeleteExamCommand extends Command {
             + PREFIX_TIME + "14:00";
 
     public static final String MESSAGE_SUCCESS = "Deleted exam from person with ID: %1$s";
-
+    private static final Logger logger = Logger.getLogger(DeleteExamCommand.class.getName());
```
Comment (MarcusGitty) [inline src/main/java/seedu/address/logic/commands/DeleteExamCommand.java:43]: Same thing here

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 3]
```diff
@@ -13,12 +16,15 @@ public class ListCommand extends ViewCommand {
     public static final String COMMAND_WORD = "view";
 
     public static final String MESSAGE_SUCCESS = "Viewing all students";
+    private static final Logger logger = Logger.getLogger(ListCommand.class.getName());
```
Comment (MarcusGitty) [inline src/main/java/seedu/address/logic/commands/ListCommand.java:19]: Same thing here

**Review by MarcusGitty — APPROVED** [review]
LGTM with just 1 comment on the Logger class name

File: src/main/java/seedu/address/logic/commands/ListCommand.java (hunk 3 above)
Comment (jessicawyz) [reply src/main/java/seedu/address/logic/commands/ListCommand.java]: Thanks for the catch! I fixed them to be the same other other classes, will merge them in.

**Pull request merged**